extensions/resteasy-reactive/rest-client-reactive-jackson/deployment/src/main/java/io/quarkus/rest/client/reactive/jackson/deployment/RestClientReactiveJacksonProcessor.java

```diff
@@ -37,6 +38,12 @@ void feature(BuildProducer<FeatureBuildItem> features) {
         features.produce(new FeatureBuildItem(REST_CLIENT_REACTIVE_JACKSON));
     }
 
+    @BuildStep
+    void nativeSupport(BuildProducer<RuntimeReinitializedClassBuildItem> runtimeReinitializedClassProducer) {
+        runtimeReinitializedClassProducer
+                .produce(new RuntimeReinitializedClassBuildItem(io.vertx.core.json.Json.class.getName()));
```

Comment:
I had no idea about `RuntimeReinitializedClassBuildItem`.
What I don't see and would like to understand is why using `io.vertx.core.json.Json.class.getName()` causes that the custom object mapper customisers to be re-init? since I think the object mapper has nothing to do with Vert.x Json :/

Reply:
`io.vertx.core.json.Json` uses Jackson under the hood. The reason ObjectMapper comes into play in native-image build time is because `io.vertx.core.json.Json` has a static field that looks up a `JsonCodec` that is backed by Jackson.

I was  able to find the root cause (`Json` class) is because I tried `RuntimeReinitializedClassBuildItem` on `ObjectMapperCustomizer` which failed (and the cause of failure could be found using `quarkus.native.additional-build-args=--trace-object-instantiation=org.acme.RegisterCustomModuleCustomizer`)

Reply:
I see, many thanks for the clarification!

Reply:
👍🏼